test/conformance/parallel/configuration_test.go

```diff
@@ -0,0 +1,27 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadConfiguration(t *testing.T) {
+	cfg := Configuration{}
+	path, _ := os.Getwd()
+	loadConfiguration(fmt.Sprintf("%s/%s", path, "testdata/cfg1.yml"), &cfg)
+	assert.Equal(t, int64(5), cfg.MasterOffsetContinuousConfig.Duration)
+	assert.Equal(t, 100, cfg.MasterOffsetContinuousConfig.MaxOffset)
```

Comment:
100 and -100 should be a variable or read from config somewhere ?

Reply:
Those variables are kept in the configuration.yaml